ipam/ipam.go

```diff
@@ -65,7 +65,7 @@ func (ipam *IPAM) Routes() common.Routes {
 
 // handleHost handles request for a specific host's info
 func (ipam *IPAM) legacyAllocateIpByName(input interface{}, ctx common.RestContext) (interface{}, error) {
```

Comment:
This function's name is now wrong. It should be changed to "legaceAllocateIpByTenantID" (or something similar).